boards/arduino-leonardo/examples/leonardo-i2cdetect.rs

```diff
@@ -2,18 +2,20 @@
 #![no_main]
 
 use arduino_leonardo::prelude::*;
+use arduino_leonardo::usart;
 use panic_halt as _;
 
 #[arduino_leonardo::entry]
 fn main() -> ! {
     let dp = arduino_leonardo::Peripherals::take().unwrap();
 
     let mut pins = arduino_leonardo::Pins::new(dp.PORTB, dp.PORTC, dp.PORTD, dp.PORTE, dp.PORTF);
+    let baudrate = 57600.into_baudrate;
     let mut serial = arduino_leonardo::Serial::new(
         dp.USART1,
         pins.d0,
         pins.d1.into_output(&mut pins.ddr),
-        57600,
+        baudrate,
     );
```

Comment:
As above, please just inline the `.into_baudrate()` call:

```suggestion
    let mut serial = arduino_leonardo::Serial::new(
        dp.USART1,
        pins.d0,
        pins.d1.into_output(&mut pins.ddr),
        57600.into_baudrate(),
    );
```